kotlin/add-graphics-renderer/README.md

```diff
@@ -14,11 +14,11 @@ Run the sample and view graphics for points, lines, and polygons, which are styl
 
 ## How it works
 
-* Create a `GraphicsOverlay` and add it to the `MapView`.
-* Create a `Graphic`, specifying only a `Geometry`.
-* Create a single `Symbol` such as a `SimpleMarkerSymbol`.
-* Create a renderer with the `Symbol` such as a `SimpleRenderer(symbol)`.
-* Set the renderer on the `GraphicsOverlay` with `graphicsOverlay.renderer = renderer`.
+1 Create a `GraphicsOverlay` and add it to the `MapView`.
```

Comment:
whoops, these need to have a period after them so they render correctly in Markdown. If you click the "display the rich diff" icon along the right of the README.md in GitHub, you can see how it will render as it is currently. 
```suggestion
1. Create a `GraphicsOverlay` and add it to the `MapView`.
```

Reply:
Wow good catch, thanks!